lib/fog/kubevirt/compute/models/services.rb

```diff
@@ -12,12 +12,16 @@ class Services < Fog::Collection
         def all(filters = {})
           begin
             srvs = service.list_services(filters)
+
+            @kind = srvs.kind
+            @resource_version = srvs.resource_version
           rescue ::Fog::Kubevirt::Errors::ClientError
             # we assume that we get 404
             srvs = []
+
+            @kind = 'Service'
           end
-          @kind = srvs.kind
-          @resource_version = srvs.resource_version
```

Comment:
NOTE I didn't set `@resource_version` here because I think `nil` is a decent "empty state" value but if I'm wrong I can certainly set this to `@resource_version = ""` or something like that.